apps/accounts/validators.py

```diff
@@ -0,0 +1,129 @@
+import datetime
+from django.core.exceptions import ValidationError
+from django.db.models import Q
+from django_password_validators.password_history.models import (
+    PasswordHistory,
+    UserPasswordHistoryConfig,
+)
```

Comment:
Since we are importing models here, will we need to run migrations for these in our cloud environments?

Reply:
there are 2 extra models for password history, so migration is needed.